lib/commands/WeatherCommand.js

```diff
@@ -0,0 +1,27 @@
+const weather = require('openweathermap-js')
+
+const Command = require('./Command')
+
+module.exports = new Command({
+	name: 'weather',
```

Comment:
It seems you are using tabs here. Can you switch to using spaces, with 4 spaces matching 1 level of indentation? This will ensure consistency across the project.